samples/apps/copilot-chat-app/webapi/Config/PromptsConfig.cs

```diff
@@ -33,6 +44,16 @@ public class PromptsConfig
 
     public void Validate()
     {
+        if (this.CompletionTokenLimit <= 0)
+        {
+            throw new ArgumentOutOfRangeException(nameof(this.CompletionTokenLimit), $"{nameof(this.CompletionTokenLimit)} is not valid: '{this.CompletionTokenLimit}' is not greater than 0.");
+        }
+
+        if (this.ResponseTokenLimit <= 0)
+        {
+            throw new ArgumentOutOfRangeException(nameof(this.ResponseTokenLimit), $"{nameof(this.ResponseTokenLimit)} is not valid: '{this.ResponseTokenLimit}' is not greater than 0.");
+        }
+
```

Comment:
Might also worth checking response limit cannot be greater than completion limit.